src/open_inwoner/templates/pages/search.html

```diff
@@ -12,23 +12,13 @@ <h1 class="utrecht-heading-1">{% trans "Zoekresultaten voor " %} "{% firstof sea
 
     {#  facets and results section #}
     <div class="grid">
-    {% if paginator.count %}
```

Comment:
If there are no query results, filters should not be visible. Not sure if you may need the {% if paginator.count %}...{% else %} block from PR https://github.com/maykinmedia/open-inwoner/pull/1638 or can solve it in this one already. Here the problem is, it is pushing the column to the side on Desktop.

Desktop:

<img width="808" alt="Screenshot 2025-03-18 at 11 12 31" src="https://github.com/user-attachments/assets/982d5ff3-5eb1-4120-b77d-f031621319a7" />

Mobile:

<img width="458" alt="Screenshot 2025-03-18 at 11 17 05" src="https://github.com/user-attachments/assets/fd615736-bda7-4ec6-990d-4c1c753e98ab" />